### ISSUE TITLE
0.5.0 release: Install on M1 Mac fails on not finding libtorrent

### ISSUE THREAD

**Issue by sgoodfriend** [Farama-Foundation/AutoROM#20]
I got the same error on Python 3.8 and 3.9. The following error prints out on Python 3.10:
```
❯ pip install AutoROM.accept-rom-license                 
Collecting AutoROM.accept-rom-license
  Using cached AutoROM.accept-rom-license-0.5.0.tar.gz (10 kB)
  Installing build dependencies ... error
  error: subprocess-exited-with-error
  
  × pip subprocess to install build dependencies did not run successfully.
  │ exit code: 1
  ╰─> [11 lines of output]
      Ignoring importlib-resources: markers 'python_version < "3.9"' don't match your environment
      Collecting setuptools
        Using cached setuptools-65.6.3-py3-none-any.whl (1.2 MB)
      Collecting wheel
        Using cached wheel-0.38.4-py3-none-any.whl (36 kB)
      Collecting requests
        Using cached requests-2.28.1-py3-none-any.whl (62 kB)
      Collecting tqdm
        Using cached tqdm-4.64.1-py2.py3-none-any.whl (78 kB)
      ERROR: Could not find a version that satisfies the requirement libtorrent (from versions: none)
      ERROR: No matching distribution found for libtorrent
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× pip subprocess to install build dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

Workaround was to install 0.42 explicitly: `pip install AutoROM.accept-rom-license==0.4.2`

**Comment by rajfly** [comment]
On Apple Silicon with the 0.5.0 release, installing `autorom` first and then accepting the license works: 
```bash
pip install autorom
AutoROM --accept-license
```

**Comment by ArturNiederfahrenhorst** [comment]
Ran into this issue when attempting to install autorom with `pip install 'autorom[accept-rom-license]'`, could resolve with @sgoodfriend 's workaround.

**Comment by jjshoots** [comment]
Hi, @sgoodfriend, may I know what pip version you're using?
Any version prior to v21 does not have libtorrent indexed, so updating pip may solve the issue.

**Comment by sgoodfriend** [comment]
@jjshoots , 22.3.1
```
❯ pip --version                                   
pip 22.3.1 from /Users/sgoodfriend/miniconda3/envs/su_imp/lib/python3.10/site-packages/pip (python 3.10)
❯ pip install --upgrade AutoROM.accept-rom-license
Requirement already satisfied: AutoROM.accept-rom-license in /Users/sgoodfriend/miniconda3/envs/su_imp/lib/python3.10/site-packages (0.4.2)
Collecting AutoROM.accept-rom-license
  Using cached AutoROM.accept-rom-license-0.5.0.tar.gz (10 kB)
  Installing build dependencies ... error
  error: subprocess-exited-with-error
  
  × pip subprocess to install build dependencies did not run successfully.
  │ exit code: 1
  ╰─> [11 lines of output]
      Ignoring importlib-resources: markers 'python_version < "3.9"' don't match your environment
      Collecting setuptools
        Using cached setuptools-65.6.3-py3-none-any.whl (1.2 MB)
      Collecting wheel
        Using cached wheel-0.38.4-py3-none-any.whl (36 kB)
      Collecting requests
        Using cached requests-2.28.1-py3-none-any.whl (62 kB)
      Collecting tqdm
        Using cached tqdm-4.64.1-py2.py3-none-any.whl (78 kB)
      ERROR: Could not find a version that satisfies the requirement libtorrent (from versions: none)
      ERROR: No matching distribution found for libtorrent
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× pip subprocess to install build dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by jjshoots** [comment]
@sgoodfriend Thanks for the report. While we continue to figure out why no version of libtorrent is found for M1, could you use autorom v0.4.2 instead for the time being?

**Comment by pseudo-rnd-thoughts** [comment]
It looks like libtorrent doesn't support apple silicon but they are looking to add it. In the meantime, you can still use v0.4.2 like @jjshoots mentioned https://github.com/arvidn/libtorrent/issues/7131

**Comment by jjshoots** [comment]
Hi everyone, all the problems should be fixed with AutoROM v0.6.0:
```
pip3 install AutoROM>=0.6.0
```
We don't recommend anyone use any version of AutoROM prior to 0.6.0 anymore.

Cheers!
